pandas/core/base.py

```diff
@@ -678,8 +678,10 @@ def _is_homogeneous_type(self):
 
         See Also
         --------
-        DataFrame._is_homogeneous_type
-        MultiIndex._is_homogeneous_type
+        DataFrame._is_homogeneous_type : Whether all the columns in a
+            DataFrame have the same dtype.
+        MultiIndex._is_homogeneous_type : Whether all the levels of a
+            MultiIndex  have the same dtype.
```

Comment:
there's an extra space between "MultiIndex" and "have"

Reply:
Amended